torch/csrc/autograd/python_function.cpp

```diff
@@ -559,6 +559,16 @@ static void _trace_post_record(
   }
 
   node->i_(jit::attr::inplace, is_inplace);
+  if (node) {
+    auto module_name = PyDict_GetItemString(((PyTypeObject*)op_obj)->tp_dict, "__module__");
+    if (module_name) {
+      const char *ptr = PyUnicode_AsUTF8(module_name);
+      if (ptr) {
+          auto modname = std::string(ptr);
+          node->s_(jit::attr::module, modname);
```

Comment:
Suggestion for slightly more compact code without loss of clarity:
```suggestion
          node->s_(jit::attr::module, std::string(ptr));
```